tests/Doctrine/Tests/OrmFunctionalTestCase.php

```diff
@@ -724,7 +732,10 @@ protected function _getEntityManager(
 
             $cacheConfig    = new CacheConfiguration();
             $cache          = $this->getSharedSecondLevelCacheDriverImpl();
-            $factory        = new DefaultCacheFactory($cacheConfig->getRegionsConfiguration(), $cache);
+
+            $regionConfig = $cacheConfig->getRegionsConfiguration();
+            $regionConfig->setDefaultLifetime(static::DEFAULT_CACHE_REGION_LIFE_TIME);
```

Comment:
Is this change really required for the test?

Reply:
For reproducing this problem,  I set second level cache time limit  = 1 second. This code need for this.

Reply:
Sorry, what I meant to say was: can't we keep this contained to the functional test only? This is now changing the default for every test in the suite, which might create undesired side-effects 😁

Reply:
> Sorry, what I meant to say was: can't we keep this contained to the functional test only? This is now changing the default for every test in the suite, which might create undesired side-effects grin

Done